src/server/api/controllers/users.controller.js

```diff
@@ -18,6 +19,26 @@ const createUser = async (body) => {
   };
 };
 
+const getUsers = async () => knex('users');
+
+const getUserById = async (id) => {
+  if (!id) {
+    throw new HttpError('Bad request. Id should be a number', 400);
+  }
+
+  try {
+    const users = await knex('users')
+      .select('users.id as id', 'full_name', 'email', 'mobile', 'address')
+      .where({ id });
+    if (users.length === 0) {
+      throw new Error(`Specified ID does not existt`, 404);
```

Comment:
Did you mean to use `HttpError` here?

Also, note that this exception will be caught by the `catch` clause on line 37.